common/views/components/Tags/Tags.js

```diff
@@ -53,6 +53,19 @@ const Tags = ({ tags }: Props) => {
                       'border-color-green border-width-1': true,
                     })}
                   >
+                    {/* An empty span for the light and medium font weights, so that
+                    the tag will always be the height of the larger of the two. */}
```

Comment:
Don't we know which one is largest, and we just have that one here?

Reply:
I don't know if we can guarantee which one would be largest for a given system setup.

Reply:
Ah, okay - so on some systems a `L` is bigger and on other `M` is bigger.
Makes you wonder about making fonts in the first place.